### DISCUSSION TITLE
My github dark theme is not the same!

### DISCUSSION THREAD

**Discussion by yasineker028** [community/community#124521] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

![Ekran görüntüsü 2024-05-18 133334](https://github.com/community/community/assets/124465818/b46c5dbc-4aa1-489b-9a28-d16f096c5ea1)
![Ekran görüntüsü 2024-05-18 133353](https://github.com/community/community/assets/124465818/d1f65b17-fa34-4482-859d-1ecd9b51c5dd)

**Comment by CopperEagle** [comment]
Hey @itsyasineker 

Where do you see these settings?

**Reply by yasineker028** [reply]
https://monaspace.githubnext.com/
and Github Dark theme on VS Code

**Reply by CopperEagle** [reply]
:confused:  I guess there's no guarantee that VS code exactly replicates the color code from Github Dark Theme.

If you wanted to really change the coloring scheme to be the same, you will probably need to do it manually.  [Here](https://github.com/StylishThemes/GitHub-Dark/blob/master/src/themes/github/github-dark.css) is the color code for GitHub Dark theme code coloring and [here](https://stackoverflow.com/questions/43986680/change-colors-in-a-visual-studio-code-color-theme) is how to adapt the color code in VS code.

**Reply by yasineker028** [reply]
I really can't adapt that code in my VS Code. I need help.

**Reply by yasineker028** [reply]
Can you tell me step by step what I should do?

**Reply by CopperEagle** [reply, marked as answer]
Sure. So I assume you installed the GitHub Dark Theme using the [official extension](https://open-vsx.org/extension/GitHub/github-vscode-theme). ([Here](https://marketplace.visualstudio.com/items?itemName=GitHub.github-vscode-theme) Marketplace.) If this is the case, you need to go to `.vscode/extensions`. Then you need to enter the extension folder, likely called "`github.github-vscode-theme-6.3.4-universal`" or similar. There is a subfolder called `themes`, enter there.

Now, there are the config files for the different themes. This includes semantic coloring. Here is the section for `dark-default.json`:
![Github Dark Theme configuration file section](https://github.com/community/community/assets/137458753/b61e7c5d-e926-4405-b7b2-03a145d1a77c)
For example, if you do not like the appearance of strings, there is a "scope" called "*string*". You can there change the color code for that. Sometimes a color is being used for several different source code elements. In this case, sope is a list. Now you can compare and adjust using the [official theme](https://github.com/StylishThemes/GitHub-Dark/blob/master/src/themes/github/github-dark.css).

Hope this helps.

*Ps. You may want to try to have AI doing the adjustment for you. They are getting pretty good at recalling. Maybe Google Gemeni is best for this thanks to its context window being 10x longer than anyone else's. But probably the 128k context window of ChatGPT is good enough.*

**Reply by yasineker028** [reply]
Hello! Thanks for helping me. I have a problem. Nothing changed and some errors.
![image](https://github.com/community/community/assets/124465818/0dd1a774-73fd-4280-b83c-e7520da34164)

**Reply by CopperEagle** [reply]
Yeah, these are different "specification formats". The official specs uses CSS. But you need to adapt the JSON. That's why I recommend using AI (Google Gemini or ChatGPT *if the context window is sufficient*) to adapt it. Give it the context that it is the github dark theme and you want to make sure it looks like on the webpage, the CSS, the JSON and tell it to adapt the JSON to match the CSS file.
